click_odoo_contrib/_dbutils.py

```diff
@@ -12,7 +13,11 @@
 def pg_connect():
     conn = odoo.sql_db.db_connect("postgres")
     cr = conn.cursor()
-    cr.autocommit(True)
+    # We are not going to use the ORM with this connection
```

Comment:
reset_config_parameters seems to use env["ir.config_parameter"].init(force=True) so is this statement about not using the ORM really true?

(Edit: first comment was on the wrong side of the diff)

Reply:
Does that use a cursor obtained from pg_connect ?

Reply:
Looked a bit closer. No it doesn't (sorry it wasn't obvious to me at first). Only if you are using `postgres` as an odoo database. (But i guess then you're in trouble anyways)